lms/templates/commerce/provider.underscore

```diff
@@ -1,14 +1,14 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%- interpolate(
+    <%- edx.HtmlUtils.interpolateHtml(
           gettext("You still need to visit the %(display_name)s website to complete the credit process."),
```

Comment:
You need to switch ``%(foo)s`` to ``{foo}`` whenever you switch from interpolate to interpolateHtml.